src/pages/ProgramsList/ProgramItemsSider.js

```diff
@@ -78,7 +80,7 @@ const ProgramsItemsSider = ({
           }
         }}
         defaultSelectedKeys={
-          cat ? [joinedMenuItems.findIndex((e) => e === cat).toString()] : ["0"]
+          cat ? [joinedMenuItems.findIndex(e => e === cat).toString()] : ["0"]
```

Comment:
nit: can we use `category` instead of `cat` to make it more readable? also maybe 'joinedMenuItem` instead of `e`. can be done later (if suitable) tho :)